### ISSUE TITLE
Does this work with YAML-Based Dashboards?

### ISSUE THREAD

**Issue by IzeQube** [daredoes/linked-lovelace-ui#22]
Hello,
this card looks really interesting, but I can't get it to work with my additional YAML based sashboards.

If I create a UI-based dashboard everything seems to work. But it isn't possible to use the template in my YAML based sashboards, they don't appear in `custom:linked-lovelace-ui` and the debug console only my UI based dashboards are listed in the array...

Does this card not work with YAML based dashboards?

Thanks.

**Comment by daredoes** [comment]
@IzeQube I may be intentionally ignoring them as I don't use any, and wanted to only change what I was certain of. 

I will look into adding support for YAML-based dashboards. 

It's actually all YAML under the hood, even if it says UI? When a user changes a card on the dashboard the front end submits THE ENTIRE DASHBOARD to the backend to be updated, just for a one character change. While the backend takes JSON, it converts all of it to and from YAML for storage, even for UI. Ideally, this means it won't be too hard to add.

**Comment by daredoes** [comment]
Yeah, I ignore yaml-based dashboards. @IzeQube  Version 1.5.0 should accept YAML dashboards. Let me know if it doesn't or if it screws yours up after transformation. 

Back up before upgrading :)

**Comment by IzeQube** [comment]
Oh, wow. Thanks for implemeting it so quick.

I'll try it this evening...

**Comment by daredoes** [comment]
@IzeQube let me know how it goes, and if it needs some debugging.

**Comment by IzeQube** [comment]
We'll, i've tried using it.
It doesn't seem to be working with my YAML files... If I can see it right, they still aren't listed in the Array in Debug Mode...

I'm setting um the my new dashboard in UI mode (und using your fantastic card), it's much better than it used to be in recent Home Assistant versions, so it isn't so important for me at the moment.

**Comment by daredoes** [comment]
Still important in the long run! 

I'll keep looking into.
